### PR TITLE
Update API Gateway example to use Mcc-Version header

### PR DESCRIPTION
**Pull request by ssteeg-mdsol**
As part of the upcoming API Gateway release, the versioning schema will change with the version information being passed in a `Mcc-Version` header instead of as part of the URL. This PR updates the example code for calling the API Gateway and does some minor style cleanup on other areas of the docs.

@vipinj @glow-mdsol @jcarres-mdsol @ykitamura-mdsol @piao-mdsol @isparks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,50 @@
...
 To use MAuth authentication you will need:
 
 * An MAuth APP ID
 * An MAuth private key (with the public key registered with Medidata's MAuth server)
 
 
-## Using ##
+## Using
+
+```python
```
Comment (piao-mdsol) [inline README.md]: 👏 👏 👏

File: README.md [hunk 2]
```diff
@@ -1,46 +1,50 @@
 # requests-mauth
 
+[![Build Status](https://travis-ci.org/mdsol/requests-mauth.svg?branch=master)](https://travis-ci.org/mdsol/requests-mauth.svg?branch=master)
```
Comment (ykitamura-mdsol) [inline README.md]: maybe better to remove `.svg?branch=master` from the link:
```
[![Build Status](https://travis-ci.org/mdsol/requests-mauth.svg?branch=master)](https://travis-ci.org/mdsol/requests-mauth)
```

File: README.md [hunk 3]
```diff
@@ -1,46 +1,50 @@
...
-    if result.status_code == 200:
-        print([r['uuid'] for r in result.json()['studies']])
-    print(result.text)
 
-Development
------------
-We use [travis](https://travisci.com) for automated CI of the code (and status checks are required to pass prior to PR merges being accepted).  
+## Development
+
+We use [travis](https://travisci.com) for automated CI of the code (and status checks are required to pass prior to PR merges being accepted).
```
Comment (ykitamura-mdsol) [inline README.md]: screen shot 2019-03-05 at 15 35 58

Need to update this URL to `https://travis-ci.com`

File: docs/examples.rst [hunk 4]
```diff
@@ -119,11 +123,18 @@ In this example we add the headers to allow the Application to impersonate a use
         :param dict configuration: a configuration dictionary
         :param str user_uuid: a User UUID per the platform
         """
-        full_url = "https://mcc.example.org/v1/service"
+        full_url = "https://mcc.example.org/service"
+        headers = {
+            "Accept": "application/json",
+            "Mcc-Version": "v2019-03-22",
+            "MCC-Impersonate": "com:mdsol:users:{}".format(user_uuid)
```
Comment (jcarres-mdsol) [inline docs/examples.rst]: we blacklist this header! Do not want them to impersonate

Comment (glow-mdsol) [reply docs/examples.rst]: Why not?  Having apps perform activities on behalf of a user seems to be one of the primary goals of initiatives like API-gateway

Comment (ssteeg-mdsol) [reply docs/examples.rst]: Updated in https://github.com/mdsol/requests-mauth/pull/14/commits/b1da514750fd6ae9a19a6f7fd64eae446b995e03 to keep the impersonation example as is for non-API Gateway services. The API Gateway Lambda is already doing impersonation on behalf of the requester

File: .travis.yml [hunk 5]
```diff
@@ -1,15 +1,21 @@
...
-  - "3.4"
-  - "3.5"
-  - "3.6"
-  - "pypy"
+matrix:
+  fast_finish: true
+jobs:
+  include:
+    - python: '2.7'
+    - python: "2.7"
```
Comment (ykitamura-mdsol) [inline .travis.yml]: @glow-mdsol `2.7` is duplicated...

Comment (ssteeg-mdsol) [reply .travis.yml]: Addressed in https://github.com/mdsol/requests-mauth/pull/17 - Thanks @glow-mdsol!

**Pull request merged**